src/test/resources/features/ppe/PPEClassificationCreate.feature

```diff
@@ -0,0 +1,12 @@
+# @Author: Walker Colina
+# feature to review behaviour of create ppe classification
+Feature: PPEClassificationCreate
+
```

Comment:
Eliminar espacio

Reply:
No changes here. The blank line is generated for the ''Reformat Code' option.